### PR TITLE
update swdc-tracker and modify the packaging of the UI element data, …

### PR DESCRIPTION
**Pull request by mmclead**
…tracking some additional clicks

### CODE REVIEW

**Comment by mmclead** [comment]
This will needs an updated swdc-tracker version that has the updated json schema version

File: src/Constants.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const CODE_TIME_EXT_ID = "softwaredotcom.swdc-vscode";
 export const CODE_TIME_TYPE = "codetime";
 
 // API ENDPOINT
-export const api_endpoint = "https://api.software.com";
+export const api_endpoint = "https://stagingapi.software.com";
```
Comment (xavluiz) [inline src/Constants.ts:13]: we should change these back to prod before publishing, thanks

Comment (mmclead) [reply src/Constants.ts]: fixed

File: src/managers/TrackerManager.ts [hunk 2]
```diff
@@ -21,84 +23,97 @@ export class TrackerManager {
...
 		if (result.status === 200) {
 			this.trackerReady = true;
 		}
 	}
 
-	/**
-	 * @param type execute_command | click
-	 * @param ui_element {element_name, element_location, color, icon_name, cta_text}
-	 */
+	public resetJwt() {
```
Comment (xavluiz) [inline src/managers/TrackerManager.ts:33]: the resetJwt should be needed. the session.json isn't cached and we should be able to always perform...
const jwt = getItem("jwt");

Comment (mmclead) [reply src/managers/TrackerManager.ts:33]: I'm wanting to cache it so we don't dig into a file every time.    Anytime we set the JWT key, I've instrumented the 'resetJWT' call.

File: src/DataController.ts [hunk 3]
```diff
@@ -170,6 +171,7 @@ export async function isLoggedIn(): Promise<boolean> {
     const state = await getUserRegistrationState();
     if (state.loggedOn) {
         initializePreferences();
+        TrackerManager.getInstance().resetJwt();
```
Comment (xavluiz) [inline src/DataController.ts:174]: lets move this to the getUserRegistrationState where we update the JWT, thanks.

**Pull request merged**